### PR TITLE
Change IBM entries according to latest guidelines

### PR DESCRIPTION
**Pull request by SNiemann15**
The IBM Z family has changed naming guidelines see this PR: https://github.com/openshift/openshift-docs/pull/61755

Also the IBM Power entry (ex p series) was very outdated. See: https://github.com/openshift/openshift-docs/pull/41204

### CODE REVIEW

**Comment by SNiemann15** [comment]
@alishaibm @lsmcfadden please take a look at the change for IBM Power

**Comment by lsmcfadden** [comment]
@SNiemann15 
I was thinking we just use the wording from the naming doc (w/o adding ppc64le architecture)?  Something like:

IBM Power® (noun)
**Description:** ~IBM Power represents the family of IBM Power processors (ppc64le architecture) and its products.~ 

IBM Power represents the family of servers based on Power processors that can include combinations of IBM Power servers, systems software and solutions. 
-or-
IBM Power represents the family of servers based on Power processors that can include combinations of IBM Power servers and its products.

**Use it****:** yes
**Incorrect forms:** pSeries, System P, Power Systems

cc @manokuma for veto

**Comment by dfitzmau** [comment]
Hi @SNiemann15 .

Would you be OK with adding an index entry for IBM Cloud Bare Metal (Classic) as part of this PR? This entry replaces IBM Cloud. 

I've a PR that updates the OpenShift docs for this term change: https://github.com/openshift/openshift-docs/pull/61782/files

WDYT, @bergerhoffer , should I create a separate Issue for the IBM Cloud Bare Metal (Classic) index or could it be bundled in this open PR?

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 1]
```diff
@@ -11,37 +11,38 @@
 
 
 [discrete]
-[[ibm-eserver-system-p]]
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:14]: Change required to the index `ISeries (noun)`?

Screenshot from 2023-07-12 12-50-55

Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:14]: And:

Screenshot from 2023-07-12 12-51-35

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 2]
```diff
@@ -11,37 +11,38 @@
...
-*Incorrect forms*: S/390, S90
+*Incorrect forms*: LinuxONE, linuxone
 
-*See also*:
+*See also*: xref:ibm-z[IBM Z]
 
 [discrete]
 [[ibm-z]]
-==== image:images/yes.png[yes] IBM Z (noun)
-*Description*: _IBM Z_ is the new name for the "IBM z Systems" family of IBM mainframe computers.
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:38]: The following index is still correct?

Screenshot from 2023-07-12 12-53-11

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 3]
```diff
@@ -11,37 +11,38 @@
 
 
 [discrete]
-[[ibm-eserver-system-p]]
-==== image:images/yes.png[yes] IBM eServer System p (noun)
-*Description*: Use "IBM eServer System p" for the first reference, and "IBM System p" or "System p" for subsequent references.
+[[ibm-power]]
+==== image:images/yes.png[yes] IBM Power® (noun)
+*Description*: _IBM Power_ represents the family of IBM Power processors (`ppc64le` architecture) and its products.
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:16]: `(`ppc64le` architecture)` is the only compatible architecture or is this just an example?

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 4]
```diff
@@ -512,13 +513,13 @@ There is no functional difference between the first server that was installed an
...
-*Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Only use in RHEL documentation. 
+*Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Only use in RHEL documentation.
 
 Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
 
-Example: 
+Example:
 
-_This feature can run on only Intel 64 processors_ 
+_This feature can run on only Intel 64 processors_
```
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:522]: Suggestion: 

`This feature can run only on Intel 64 processors._`

The placement of 'only' reads strangely as is. I think the guidelines about placement of 'only' would only really be an issue if it came before the verb so I think it's better placed before the preposition in this case. 
*Others might disagree.

There's also a missing period at the end of the sentence.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 5]
```diff
@@ -512,13 +513,13 @@ There is no functional difference between the first server that was installed an
 [discrete]
 [[intel-64]]
 ==== image:images/yes.png[yes] Intel 64 (noun)
-*Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Only use in RHEL documentation. 
+*Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Only use in RHEL documentation.
```
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:516]: I left another 'only' comment below but here I think it should be:
`Use only in RHEL documentation.`

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc (hunk 4 above)
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:522]: @ndeevy "only" is used correctly. "Only" is a modifier that's very easy to misplace. For example, "This feature can only run on Intel 64 processors" might sound better, but it means "the only thing this feature can do is run on Intel 64 processors", which isn't the case. With the current use of "only", the sentence means "The only thing the feature can run on is Intel 64 processors", which is indeed the correct meaning.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc (hunk 5 above)
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:516]: Agree

**Comment by ndeevy** [comment]
@SNiemann15  I left just a couple of tiny comments/queries. Otherwise, this PR seems good. 
Thanks

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc (hunk 4 above)
Comment (ndeevy) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:522]: Got that @mportman12, TY :-] my suggestion was just to move it one space back to before 'on' (not 'only run on', which _would_ be an issue). `This feature can run only on Intel 64 processors.` still means "The only thing the feature can run on is Intel 64 processors" but, I support your point so can stand down my comment. :-]

Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:522]: @ndeevy ah got it! Sorry for the confusion.

**Comment by SNiemann15** [comment]
> Hi @SNiemann15 .
> 
> Would you be OK with adding an index entry for IBM Cloud Bare Metal (Classic) as part of this PR? This entry replaces IBM Cloud.
> 
> I've a PR that updates the OpenShift docs for this term change: https://github.com/openshift/openshift-docs/pull/61782/files
> 
> WDYT, @redhat-documentation/ccs-style-council, should I create a separate Issue for the IBM Cloud Bare Metal (Classic) index or could it be bundled in this open PR?

@dfitzmau I'm ok with adding that to my PR

**Comment by dfitzmau** [comment]
> > Hi @SNiemann15 .
> > Would you be OK with adding an index entry for IBM Cloud Bare Metal (Classic) as part of this PR? This entry replaces IBM Cloud.
> > I've a PR that updates the OpenShift docs for this term change: https://github.com/openshift/openshift-docs/pull/61782/files
> > WDYT, @redhat-documentation/ccs-style-council, should I create a separate Issue for the IBM Cloud Bare Metal (Classic) index or could it be bundled in this open PR?
> 
> @dfitzmau I'm ok with adding that to my PR

Thanks, @SNiemann15 .

I'll let the @redhat-documentation/ccs-style-council weight in on leaping an additional index entry into an existing open PR.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc (hunk 4 above)
Comment (SNiemann15) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:522]: @ndeevy @mportman12 Oh yes "only" is tricky. I had to read your comments twice to understand 😆 but now I got it and will change accordingly.

**Comment by dfitzmau** [comment]
> Hi @SNiemann15 .
> 
> Would you be OK with adding an index entry for IBM Cloud Bare Metal (Classic) as part of this PR? This entry replaces IBM Cloud.
> 
> I've a PR that updates the OpenShift docs for this term change: https://github.com/openshift/openshift-docs/pull/61782/files
> 
> WDYT, @redhat-documentation/ccs-style-council, should I create a separate Issue for the IBM Cloud Bare Metal (Classic) index or could it be bundled in this open PR?

I'll drop this request because my PR is not merged yet. I can update the SG accordingly thereafter. https://github.com/openshift/openshift-docs/pull/61782

**Review by ndeevy — DISMISSED** [review]

**Comment by ndeevy** [comment]
LGTM so approved but I can re approve if other commits are added. TY @SNiemann15 ++

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc (hunk 1 above)
Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:14]: I see the iSeries entry has been removed. The pSeries needs removal also?

Comment (SNiemann15) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:14]: @lsmcfadden @manokuma can one of you please confirm? I would say yes it needs to be removed.

Comment (lsmcfadden) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:14]: Do you mean that you'll change it to  ~ _IBM Power_ represents the family of servers based on Power processors that can include combinations of IBM Power servers and its products which I mention above or 'remove' it altogether?

Comment (SNiemann15) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:14]: @lsmcfadden I added the IBM Power entry under i in the RH SSG glossary replacing the IBM eserver system p entry [here](https://redhat-documentation.github.io/supplementary-style-guide/#_i). 
Therefore I would remove the pSeries entry entirely unless you object. (rebranding was back in 2008 and I haven't come across any entries in RH docs)

Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:14]: I see that pSeries has been removed. Thank you.

**Review by dfitzmau — DISMISSED** [review]

**Review by ndeevy — DISMISSED** [review]

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 6]
```diff
@@ -11,37 +11,38 @@
...
+*Incorrect forms*: LinuxONE, linuxone
 
-*See also*:
+*See also*: xref:ibm-z[IBM Z]
 
 [discrete]
 [[ibm-z]]
-==== image:images/yes.png[yes] IBM Z (noun)
-*Description*: _IBM Z_ is the new name for the "IBM z Systems" family of IBM mainframe computers.
+==== image:images/yes.png[yes] IBM Z® (noun)
```
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:38]:
```suggestion
==== image:images/yes.png[yes] IBM Z (noun)
```
Remove the trademark - we don't want writers to use the trademark in product docs.

**Review by mportman12 — CHANGES_REQUESTED** [review]

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc (hunk 6 above)
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc]: I am now noticing there are trademark icons on several terms. We don't want to include the trademark unless we want writers to use the trademark in documentation - these glossary entries are instructing writers on how to spell these terms correctly, including the trademark icon. Please review the list of terms and make sure trademark icons are included only when that applies (if it ever does? Do we ever want writers to use trademark icons in product docs?).

Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc]: Hi @mportman12 . In the OCP docs we do use trademark icons on IBM components. See https://docs.openshift.com/container-platform/4.13/installing/installing_ibm_z/preparing-to-install-on-ibm-z.html . Usually on first reference of an IBM technology name.

Comment (SNiemann15) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc]: For IBM® LinuxONE there's this guidance to always use the trademark with LinuxONE: 

`Effective November 15, 2022, IBM® must precede LinuxONE in all situations`

Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc]: @lahinson ^^ regarding https://github.com/openshift/openshift-docs/pull/61782#discussion_r1265438957

**Review by dfitzmau — APPROVED** [review]

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 7]
```diff
@@ -11,37 +11,37 @@
 
 
 [discrete]
-[[ibm-eserver-system-p]]
-==== image:images/yes.png[yes] IBM eServer System p (noun)
-*Description*: Use "IBM eServer System p" for the first reference, and "IBM System p" or "System p" for subsequent references.
+[[ibm-power]]
+==== image:images/yes.png[yes] IBM Power® (noun)
+*Description*: _IBM Power_ represents the family of servers based on Power processors that can include combinations of IBM Power servers and its products. On first use, mark IBM Power with a (®).
```
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:16]:
```suggestion
*Description*: _IBM Power_ represents the family of servers based on Power processors that can include combinations of IBM Power servers and its products. On first use, mark "IBM Power" with a trademark icon (®).
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 8]
```diff
@@ -11,37 +11,37 @@
...
 
 *See also*:
 
 [discrete]
-[[ibm-s-390]]
-==== image:images/yes.png[yes] IBM S/390 (noun)
-*Description*: The _IBM S/390_ is the IBM large server (or mainframe) line of computer systems. Use the full description "IBM S/390".
+[[ibm-linuxone]]
+==== image:images/yes.png[yes] IBM® LinuxONE (noun)
+*Description*: _IBM® LinuxONE_ represents the IBM® LinuxONE platform (`s390x` architecture) and its products. IBM® must always precede LinuxONE in any context.
```
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:27]:
```suggestion
*Description*: _IBM® LinuxONE_ represents the IBM® LinuxONE platform (`s390x` architecture) and its products. "IBM®" must always precede "LinuxONE" in any context.
```

**Review by mportman12 — CHANGES_REQUESTED** [review]
Formatting and minor fixes,

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc (hunk 7 above)
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:16]: @ndeevy yes it should, good call.

Comment (ndeevy) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:16]: Ok, thanks Miriam!
 FYI @SNiemann15 that means there's a second one that needs `registered` added to Miriam's suggested `trademark icon` on line 38. Thanks :-]

**Review by ndeevy — DISMISSED** [review]

**Review by mportman12 — DISMISSED** [review]
LGTM

**Review by joaedwar — DISMISSED** [review]
lgtm

**Review by joaedwar — APPROVED** [review]

**Pull request merged**